src/strings/csv/en.csv

```diff
@@ -402,6 +402,7 @@ DESTINATION,Destination,
 DESTINATION_REQUIRED,Destination *,
 DETAILED_SITE,Detailed Site,
 DETAILS,Details,
+DEV_SERVER_ERROR,Server unavailable (may be restarting with new code changes).,
```

Comment:
Maybe add "Reload?" to the end of this so it's clear what confirming the popup will do.